launchers/install-full-deps

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sudo apt install -yy gstreamer1.0-vaapi gstreamer1.0-tools vainfo mpv python3-tinydb python3-jinja2 \
+                     wl-clipboard gnome-screensaver gnome-screenshot ffmpeg
+
+# kivu team PPA
+sudo apt-add-repository -y 'ppa:kivuteam/ppa'
+sudo apt update
+
+# ydotool
+sudo apt install -yy ydotool ydotoold
+
+# gfxi
+sudo apt install -yy gfxi
```

Comment:
Same question about -yy here

Reply:
that is a typo from an old habit, thanks